src/OOP/Animal.cs

```diff
@@ -29,18 +29,13 @@ public abstract class Animal : IMove
         protected readonly List<MovementType> MovementTypes = new List<MovementType>();
```

Comment:
Cum introducem comportamente comune in clase care nu sunt neaparat inrudite?
Un strut: e pasare, dar nu zboara.
o insecta, zboara dar nu e pasare,
Nu folosim enums ca sa introducem comportamente, folosim Interfete